README.md

```diff
@@ -60,6 +60,18 @@ app.intent(null, 'Hello Alexa what is in my calendar for today', () => 'Your cal
 app.intent('AnotherIntent', ['Hello', 'Hi', 'Whats up'], () => 'Hello yourself');
 ```
 
+### Create Welcome Message
+
+If you want more than just a generic "Welcome" from Alexa, you can use the onStart method to help you achieve that.
+
+```javascript
+app.onStart(() => {
```

Comment:
You can simplify this call by using:

```javascript
app.onStart(() => {
    return 'Welcome to My Hello World App, say hello world to get started, or say help to get more instructions';
});
```

You don't have to return the whole `responseObject`. Alexia detects it is a string and wraps it appropriately.
Also use single quotes `'` for strings (just for consistency) since this is the eslint rule we are using right now:

```
"quotes": [
    2,
    "single"
]
```